src/utils/env.ts

```diff
@@ -12,7 +12,8 @@ const buildPcEnv = (tld: string): PcEnv => ({
 });
 
 const DEFAULT_ENV_REGION = 'us-east-1';
-const environments = getEnvironments({ env: ['prod', 'fedramp'], status: ['beta', 'stable'] });
+const ENVIRONMENT_FILTERS: EnvironmentFilters = { env: ['prod', 'fedramp'], status: ['beta', 'stable'] };
```

Comment:
nit: i dont see any other usages of ENVIRONMENT_FILTERS or the EnvironmentFilters type.  If you moved the filters back inline into `getEnvironments`, then ts will infer the type and you wouldn't need to import it.

not hurting anything of course.